endtoend/endtoend_test.go

```diff
@@ -118,6 +119,22 @@ func runEndToEndTest(t *testing.T, config *e2etypes.E2EConfig) {
 		}
 	}
 
+	simulatorOutFile, err := os.Open(path.Join(e2e.TestParams.LogPath, e2e.SlasherSimulatorLogFileName))
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Run("slasher simulator completed", func(t *testing.T) {
+		if err = helpers.WaitForTextInFile(simulatorOutFile, "Correctly detected simulated attester slashing"); err != nil {
+			t.Fatalf("could not find detection logs for slasher, this means it had issues with detection: %v", err)
+		}
+		if err = helpers.WaitForTextInFile(simulatorOutFile, "Correctly detected simulated proposer slashing"); err != nil {
+			t.Fatalf("could not find detection logs for slasher, this means it had issues with detection: %v", err)
+		}
+		if err = helpers.CheckTextNotInFile(simulatorOutFile, "Did not detect"); err != nil {
```

Comment:
We can also check for the word ERROR right?

Reply:
Ah I'll add the errors here to the finishing logs, I'll leave the checks as is though